CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * [CHANGE] Flag `-azure.msi-resource` is now ignored, and will be removed in Mimir 2.7. This setting is now made automatically by Azure. #2682
 * [CHANGE] Experimental flag `-blocks-storage.tsdb.out-of-order-capacity-min` has been removed. #3261
+* [CHANGE] Removed the `Querier > Stages` panel from the `Mimir / Queries` dashboard. #3311
```

Comment:
There's a `### Mixin` section further down in this file. I think this entry belongs there

Reply:
Oops! Fixed.